tests/unit/transform/date-test.js

```diff
@@ -46,9 +46,19 @@ test("#deserialize with different offset formats", function(assert) {
   var dateStringColon = '2013-03-15T23:22:00.000+00:00';
   var dateStringShortOffset = '2016-12-02T17:30:00.000+00';
 
+  assert.expect(6);
+
+  var _dateUTC = Date.UTC;
```

Comment:
we should wrap this in a `try`/`finally` so that the original Date.UTC always gets restored.